opennms-container/minion/CONFD_README.md

```diff
@@ -136,12 +136,12 @@ Config specified will be written to `etc/org.opennms.netmgt.trapd.cfg`.
 --- 
 system:
     properties:
-        jaeger-agent-host: "<host>"
+        JAEGER_AGENT_HOST: "<host>"
```

Comment:
is there a way to not have to change this, so existing confd configs wouldn't have to be modified?

Reply:
AFAICT, any existing configs for Jaeger tracing in the Minion aren't working, because the library expects the form `JAEGER_AGENT_HOST` for system properties, not `jaeger-agent-host`, so they'll need to change anyway to work.

Reply:
I just confirmed with `opennms/minion:30.0.4`: tracing data **does not** get emitted if you just specify `jaeger-agent-host`:
```
system:
  properties:
    jaeger-agent-host: "host.docker.internal"
```

But the library does get initialized properly (but without a place to export to, since it is expecting `JAEGER_AGENT_HOST` or `JAEGER_ENDPOINT` system properties):
```
2022-12-11T22:24:14,134 | INFO  | Blueprint Extender: 1 | JaegerTracerWrapper              | 333 - org.opennms.core.tracing.jaeger-tracer - 30.0.4 | Jaeger tracer initialized with serviceName = MINION@2ffa1e5a-e64a-4943-a71d-1e8ee29e31d8
```

But, if you specify both, like this:
```
system:
  properties:
    jaeger-agent-host: "host.docker.internal"
    JAEGER_AGENT_HOST: "host.docker.internal"
```

Trace data **does** get exported. And, of course, the tracing library obviously is initialized properly:
```
2022-12-11T22:27:07,787 | INFO  | Blueprint Extender: 3 | JaegerTracerWrapper              | 333 - org.opennms.core.tracing.jaeger-tracer - 30.0.4 | Jaeger tracer initialized with serviceName = MINION@2ffa1e5a-e64a-4943-a71d-1e8ee29e31d8
```

If you just specify the `JAEGER_AGENT_HOST` system property that the tracing library expects:
```
system:
  properties:
    JAEGER_AGENT_HOST: "host.docker.internal"
```

You get nothing because the `confd` templates in 30.0.4 look for `jaeger-agent-host` in order to setup the other bits needed for tracing. There is no log showing the tracing library is initialized.